### DISCUSSION TITLE
authentication payload

### DISCUSSION THREAD

**Discussion by odecroupette** [Palakis/obs-websocket#814] [General]
Hi, could you provide a sample json payload structure required to get authenticated? thanks a lot

**Comment by tt2468** [comment]
For 4.x, you can see the authentication process described here: https://github.com/Palakis/obs-websocket/blob/4.x-current/docs/generated/protocol.md#authentication

For 5.x, it's described here: https://github.com/Palakis/obs-websocket/blob/master/docs/generated/protocol.md#creating-an-authentication-string
